docs/functions.md

```diff
@@ -0,0 +1,73 @@
+# Function Introduction
+
+## Relating with VCS tools
+
+Cyclone has been integrated with a variety of VCS tools, such as git, svn, etc. After OAuth, it can pull codes from repository and create webhook. Whenever the user commit, submit a pull request or release a version to the repository, the webhook will trigger the CI/CD workflow. 
+
+- Creating a service relating with the VCS tools
+
+<div align="center">
+	<img src="./create_service.png" alt="create_service" width="500">
+</div>
+
+- Listing all services
+
+<div align="center">
+	<img src="./list_services.png" alt="list_services" width="500">
+</div>
+
+## Continus integration and security scanning
+
+All of the processes in workflow are visible. "prebuild" compiles the executable files. "build" builds the published image. "integration" executes the integrated test. "publish" publishes the image and scans the vulnerabilities. "post build" does some relating operations after image published. "deploy" uses the published image deploy a application to Kubernetes or any other container cloud platform. Cyclone can send email to notify the result of the workflow. All of the processes are shipped by container. It will wipe off the differences cause by environment. 
```

Comment:
s/'uses the published image deploy a application to'/'uses the published image to deploy or update a application into'